src/DropdownMenu/DropdownMenu.story.tsx

```diff
@@ -93,6 +93,35 @@ SetToDefaultOpen.story = {
   name: "set to defaultOpen",
 };
 
+export const WithSubmenu = () => (
+  <DropdownMenu openAriaLabel="open dropdown" closeAriaLabel="close dropdown">
+    <DropdownLink href="/never_been">Dropdown Link</DropdownLink>
+    <DropdownButton onClick={() => {}}>Dropdown Button</DropdownButton>
+    <DropdownMenu
+      trigger={() => (
+        <DropdownButton>
+          <Flex justifyContent="space-between">
+            <Text>Submenu</Text>
+            <Text>&gt;</Text>
```

Comment:
Can we change this to an icon instead?
```suggestion
           <Icon icon="rightArrow" title="right arrow" />;
```